bears/java/CheckstyleBear.py

```diff
@@ -11,7 +11,7 @@
     'android-check-hard': 'https://raw.githubusercontent.com/noveogroup/' +
     'android-check/master/android-check-plugin/src/main/resources/' +
     'checkstyle/checkstyle-hard.xml',
-    'geosoft': 'http://geosoft.no/development/geosoft_checks.xml'}
+}
```

Comment:
I've always hated it as well, but I vaguely remember reading about it somewhere.